terraform/aws/analytical-platform-data-production/airflow/buckets.tf

```diff
@@ -1,6 +1,22 @@
+#tfsec:ignore:AVD-AWS-0089: S3 bucket logging not required
+#tfsec:ignore:AVD-AWS-0090: S3 bucket versioning not required
 resource "aws_s3_bucket" "mojap_airflow_dev" {
+  #checkov:skip=CKV_AWS_18: access logging not required
+  #checkov:skip=CKV2_AWS_62: event notification not required
+  #checkov:skip=CKV_AWS_144: cross-region replication not required
+  #checkov:skip=CKV2_AWS_61: bucket lifecycle configuration  not required
+  #checkov:skip=CKV_AWS_21: S3 bucket have versioning enabled not required
   bucket = "mojap-airflow-dev"
 
+  server_side_encryption_configuration {
+    rule {
+      apply_server_side_encryption_by_default {
+        sse_algorithm     = "aws:kms"
+        kms_master_key_id = aws_kms_key.s3_mojap_airflow_dev.arn
+      }
+    }
+  }
+
 }
```

Comment:
Had something like this come up with Ant on some of the SQS queues. The actual state of the asset is that it is already encrypted, but with `Amazon S3 managed keys (SSE-S3)`. As such I think the change we'd want is just to update the terraform to reflect that rather than re-encrypting with a customer managed key.

Reply:
yes - recommendation was to skip